pom.xml

```diff
@@ -27,7 +27,7 @@
     <groupId>org.wso2.ei</groupId>
     <artifactId>module-mongo</artifactId>
     <packaging>pom</packaging>
-    <version>0.10.0</version>
+    <version>0.9.0</version>
```

Comment:
why you downgrade the version?

Reply:
The latest release is only 0.8.2, it seems it was mistakenly bumped to 0.10 in a past PR.